flow/Makefile

```diff
@@ -303,6 +303,8 @@ OPENROAD_GUI_CMD         = $(OPENROAD_EXE) -gui $(OR_ARGS)
 
 YOSYS_CMD               ?= $(abspath $(FLOW_HOME)/../tools/install/yosys/bin/yosys)
 
+export EQY_EXE ?= $(abspath $(FLOW_HOME)/../dependencies/bin/eqy)
```

Comment:
This will fail if eqy is in the path but not at this location.

Reply:
So how can this be made to work w ORFS? Should eqy be in tools/install?

Reply:
We don't currently demand it be anywhere besides in your path.  Why do we need to narrow that?  For example I often take it from /usr/local/bin/eqy

Reply:
I thought it would be consistent w openroad. Some particular version of eqy is required for a version of ORFS, no?